learn/getting-started/building-a-data-service-in-ballerina.md

```diff
@@ -85,13 +85,13 @@ public type Employee record {|
 |};
 ```
 
-This record-type is the basis for interacting with the database.
+This record type is the basis for interacting with the database.
 
 ## Connecting to and Interacting with the Database
 
 ### Adding the MySQL Driver
 
-The MySQL driver JAR is necessary to connect to and interact with a MySQL server. There are several methods of doing this.
+The MySQL driver JAR is necessary to connect to and there are several methods to interact with a MySQL server.
```

Comment:
This makes it sound as if the examples below detail to *interact* with the MySQL database, rather than how to add the driver jar iMO

Reply:
Updated in 7e73c5565f0163ca5e5ae9de0616e88c14954143